src/components/__snapshots__/SurveyDateForm.test.tsx.snap

```diff
@@ -119,10 +119,10 @@ Object {
           <input
             class="input input--text input-type__input "
             id="Date"
-            initial_value="1997-12-24"
+            initial_value="24-12-1997"
```

Comment:
This doesn't feel like a good change. Computers get confuse with this format. Is the `02/07/2022` and English date or an Americanish one?